src/main/java/com/spotify/confidence/Confidence.java

```diff
@@ -80,31 +91,35 @@ public static Confidence.Builder builder(String clientSecret) {
 
   public static class Builder {
     private final String clientSecret;
-    private ManagedChannel managedChannel;
+    private ManagedChannel flagResolverManagedChannel =
+        ManagedChannelBuilder.forAddress("edge-grpc.spotify.com", 443).build();
 
     public Builder(@Nonnull String clientSecret) {
       this.clientSecret = clientSecret;
     }
 
-    public Builder managedChannel(String host, int port) {
-      this.managedChannel = ManagedChannelBuilder.forAddress(host, port).usePlaintext().build();
+    public Builder flagResolverManagedChannel(String host, int port) {
+      this.flagResolverManagedChannel =
+          ManagedChannelBuilder.forAddress(host, port).usePlaintext().build();
       return this;
     }
 
-    public Builder managedChannel(ManagedChannel managedChannel) {
-      this.managedChannel = managedChannel;
+    public Builder flagResolverManagedChannel(ManagedChannel managedChannel) {
```

Comment:
is there any reason we don't share the managed channel for sending events and flag resolve? as far as I know managedChannels are thread safe and can handle requests concurrently. is it the best practice in this case to not share them?

Reply:
Good point. What we want is to have the possibility to configure the ManagedChannels independent of one another. But I will make sure to use the same channel if they are configured with the same endpoint.